backend/src/graphql/resolver/ContributionResolver.test.ts

```diff
@@ -718,6 +719,28 @@ describe('ContributionResolver', () => {
         resetToken()
       })
 
+      it('throws an error with non existing filter', async () => {
+        await expect(
+          query({
+            query: listAllContributions,
+            variables: {
+              currentPage: 1,
+              pageSize: 25,
+              order: 'DESC',
+              statusFilter: ['CONFIRMD'],
```

Comment:
```suggestion
              statusFilter: ['NOT_VALID'],
```
And the exception as expetced